packages/contracts/contracts/L1/messaging/IL1CrossDomainMessenger.sol

```diff
@@ -1,6 +1,5 @@
 // SPDX-License-Identifier: MIT
-pragma solidity >0.5.0 <0.8.0;
-pragma experimental ABIEncoderV2;
+pragma solidity ^0.8.7;
```

Comment:
I think this should have the same pragma as `IL1ERC20Bridge.sol` (ie.  `pragma solidity >0.5.0 <0.9.0`) to allow for use in external packages.